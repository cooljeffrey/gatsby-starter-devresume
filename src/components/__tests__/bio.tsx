import React from "react";
import renderer from "react-test-renderer";
import Bio from "../bio";
describe("Bio", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Bio
          avatar="/static/randomstring/simon.png"
          name="test"
          summary="long summary ...."
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
