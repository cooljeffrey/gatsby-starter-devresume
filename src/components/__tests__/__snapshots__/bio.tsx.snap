// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bio renders correctly 1`] = `
<div
  className="resume-intro py-3"
>
  <div
    className="media flex-column flex-md-row align-items-center"
  >
    <img
      alt="test"
      className="resume-profile-image mb-3 mb-md-0 mr-md-5 ml-md-0 rounded mx-auto"
      src="/static/randomstring/simon.png"
      title="test"
    />
    <div
      className="media-body text-left"
    >
      <div
        className="mb-0"
        dangerouslySetInnerHTML={
          Object {
            "__html": "long summary ....",
          }
        }
      />
    </div>
  </div>
</div>
`;
